shell/actions/order_assign-guestorders2customer.php

```diff
@@ -0,0 +1,83 @@
+<?php 
+
+require_once 'public_html/app/Mage.php';
+umask(0); 
+Mage::init();
+
+function getInput($minLength = '1')
+{
+	$handle = fopen("php://stdin", "r");
+	$line = fgets($handle);
+	if (strlen(trim($line)) < $minLength) {
+            echo "error - ABORTING!\n";
+	    exit;
+	}	
+	fclose($handle);
+
+	return trim($line);
+}
+
+function getStoreViewList()
+{
+    $_store = array();
+    foreach (Mage::app()->getWebsites() as $website) {
+        foreach ($website->getGroups() as $group) {
+            $stores = $group->getStores();
+            foreach ($stores as $store) {
+                $_store[] = array("id" => $store->getWebsiteId(), "name" => $store->getName());
+            }
+        }
+    }
+	
+    return $_store;
+}
+
+// Print stores list
+$allStoresArray = getStoreViewList();
+foreach($allStoresArray as $store)
+{
```

Comment:
Following existing convention:
```suggestion 
foreach($allStoresArray as $store) {
```